### DISCUSSION TITLE
Which API key should be used for Algolia?

### DISCUSSION THREAD

**Discussion by abeldotam** [vuejs/vitepress#3698] [Q&A] [answered]
Hi folks 👋
Hope you are alright, I've a small question.

I would like to implement Algolia Search on my Vitepress's website, but the documentation is a bit synthetic ([link](https://vitepress.dev/reference/default-theme-search#algolia-search))

<img width="895" alt="image" src="https://github.com/vuejs/vitepress/assets/5216201/06ae0229-ec48-4819-a979-255075fd3c07">

That's the only guideline for setting up this, and on Algolia I've this: no less than 4 API key.

<img width="1785" alt="Capture d’écran 2024-03-25 à 09 05 45" src="https://github.com/vuejs/vitepress/assets/5216201/8f6534ab-b6a6-446c-b735-41b2a095ad17">

--- 

My question is: which one should I use?
- The Search-only?
- The Admin key?
- The Usage key?
- The monitoring key? (Doubt, for this one)

Also, is there any mandatory step for implementing Algolia?

I wasn't sure if that's an issue (so I created a discussion), but if it is, that's a lack of documentation and maybe the answers here could help the next ones who are wondering also.

Cheers

**Comment by brc-dd** [comment, marked as answer]
search only - public API key to use in your frontend code

> Also, is there any mandatory step for implementing Algolia?

Likely no. The provided crawler config should work fine.

**Reply by abeldotam** [reply]
Awesome, thanks!

It seems to work for the connexion.
From what I can see, the crawler will do it jobs tomorrow, hope it'll works straight out of the box 🤞

In anycase, thanks for the awesome work and the quick answer.
I wish you a very good day (you can close the issue if that's good for you). 🙂
